ftplugin/cue/splitjoin.vim

```diff
@@ -0,0 +1,18 @@
+if !exists('b:splitjoin_split_callbacks')
+      let b:splitjoin_split_callbacks = [
+            \ 'sj#cue#SplitStructLiteral',
+            \ 'sj#cue#SplitArray',
+            \ 'sj#cue#SplitArgs',
+            \ ]
+endif
+
+if !exists('b:splitjoin_join_callbacks')
+      let b:splitjoin_join_callbacks = [
+            \ 'sj#cue#JoinStructLiteral',
+            \ 'sj#cue#JoinArray',
+            \ 'sj#cue#JoinArgs',
+            \ ]
+endif
+
+" in CUE, trailing comma means something else
+let b:splitjoin_trailing_comma = 0
```

Comment:
Instead of setting the variable, you could remove the check for it in the code. Or is that variable used in settings by one of the parsers or something?